src/dali/plugin/input/rest/binance_com.py

```diff
@@ -0,0 +1,940 @@
+# Copyright 2022 macanudo527
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Binance.com REST plugin links:
+# REST API: https://binance-docs.github.io/apidocs/
+# Authentication: https://binance-docs.github.io/apidocs/spot/en/#introduction
+# Endpoint: https://api.binance.com
+
+# CCXT documentation:
+# https://docs.ccxt.com/en/latest/index.html
+
+import datetime as dt

```

Comment:
Since you're alredy importing from datetime at line 26, just remove this line and add the element you need there (see my comment below).